tests/operators/test_airflow_async.py

```diff
@@ -28,7 +28,7 @@
 from cosmos.profiles import get_automatic_profile_mapping
 
 DBT_PROJECTS_ROOT_DIR = Path(__file__).parent.parent.parent / "dev/dags/dbt"
-DBT_PROJECT_NAME = "original_jaffle_shop"
+DBT_PROJECT_NAME = "jaffle_shop"
```

Comment:
Apart from file deletions by removing the `dbt/original_jaffle_shop` folder in the PR, this is second of the two changes we're introducing in this PR. Adding this comment for helping reviewers.